src/systems/systems/src/pressurization/mod.rs

```diff
@@ -30,26 +58,96 @@ impl Pressurization {
         }
 
         Self {
+            cabin_pressure_simulation: CabinPressure::new(),
             cpc: [CabinPressureController::new(); 2],
-            outflow_valve: PressureValve::new(),
+            outflow_valve: PressureValve::new_outflow_valve(),
+            safety_valve: PressureValve::new_safety_valve(),
+            residual_pressure_controller: ResidualPressureController::new(),
             active_system: active,
             landing_elevation: Length::new::<foot>(0.),
+            departure_elevation: Length::new::<foot>(0.),
             sea_level_pressure: Pressure::new::<hectopascal>(1013.25),
             destination_qnh: Pressure::new::<hectopascal>(0.),
+            is_in_man_mode: false,
+            man_mode_duration: Duration::from_secs(0),
+            packs_are_on: true,
+            lgciu_gear_compressed: true,
         }
     }
 
-    pub fn update(&mut self, context: &UpdateContext, engines: [&impl EngineCorrectedN1; 2]) {
+    pub fn update(
+        &mut self,
+        context: &UpdateContext,
+        press_overhead: &PressurizationOverheadPanel,
+        engines: [&impl EngineCorrectedN1; 2],
+        lgciu: [&impl LgciuWeightOnWheels; 2],
+    ) {
+        self.cabin_pressure_simulation.update(
+            context,
+            self.outflow_valve.open_amount(),
+            self.safety_valve.open_amount(),
+            self.packs_are_on,
+            self.lgciu_gear_compressed,
+            self.cpc[self.active_system - 1].is_ground(),
+            self.cpc[self.active_system - 1].should_open_outflow_valve()
+                && !press_overhead.is_in_man_mode(),
+        );
+
+        if !press_overhead.ldg_elev_is_auto() {
+            self.landing_elevation = Length::new::<foot>(press_overhead.ldg_elev_knob.value())
+        }
+        self.lgciu_gear_compressed = lgciu
+            .iter()
+            .all(|&a| a.left_and_right_gear_compressed(true));
```

Comment:
**thought**: We're likely to start repeating code like this. Currently there's no overarching type to query both LGCIUs. Maybe this type should be added, maybe not. I'm curious what @crocket63 and @beheh think.

I'm seeing that on the hydraulics side both LGCIUs are passed separately, as each individual LGCIU ends up in some controller. It also has code like this which basically calculates the same thing:
```rust
        let is_in_flight_gear_lever_up = !(lgciu1.left_and_right_gear_compressed(true)
            || lgciu2.left_and_right_gear_compressed(true)
            || self.is_gear_lever_down);
```

Reply:
The way I understand the documentation, the CPCs get a discrete signal from each LGCIUs, then compare them and throw a mantenance error if they missmatch. Thinking now about the logic to set `self.lgciu_gear_compressed`, potentially it should check for `.any()` instead of `.all()`. But I don't know enough about how the LGCIUs work unfortunately, maybe @crocket63 can help? 

Rewording: what would we want to happen if both LGCIUs are sending different signals for compression? Should the aircraft assume it's on the ground, or still in the air?